files/build_templates/init_cfg.json.j2

```diff
@@ -177,6 +177,12 @@
             "num_dumps": "3"
         }
     },
+{%- if include_p4rt == "y" %}
+    "NODE_CFG": {
+        "integrated_circuit0": {
+            "node-id": "1"
```

Comment:
I'm not sure it makes sense to use a hard coded value here. In practice we would want this value to be unique per device so that the controller has some confidence that it's talking to the correct machine.

Reply:
Removed the hardcoded value in this file.

Reply:
Should we add README in https://github.com/sonic-net/sonic-pins to provide the instruction about p4rt functionality and usage?

So far we have seen community asking relevant questions https://github.com/sonic-net/sonic-buildimage/issues/19589 and https://github.com/sonic-net/sonic-buildimage/issues/19328